README.md

```diff
@@ -119,7 +119,7 @@ Additionally, as part of this change the existing v3.x extension has been moved
 
 | Version |  Application Insights | Angular              | Branch
 |---------|-----------------------|----------------------|-----------
-| 14.0.0  | TBD (^3.x)            | peer: ^14.0.3        | [main](https://github.com/microsoft/applicationinsights-angularplugin-js)
+| 14.0.0  | ^3.0.2                | peer: ^14.0.3        | [main](https://github.com/microsoft/applicationinsights-angularplugin-js)
```

Comment:
As with the react-native, lets move this forward to a minimum of 3.0.3, so this will also mean updating the package.json to reflect the same.